Makefile

```diff
@@ -704,6 +704,31 @@ clean-e2e-inst-signatures:
 #
 	$(CMD_RM) -rf $(OUTPUT_DIR)/e2e-inst-signatures
 
+
+AWK_CMD_TEST_COLOR = '{ \
```

Comment:
![image](https://github.com/aquasecurity/tracee/assets/3372117/e3fb507f-5e0d-454a-b4bb-3aeac92ea56f)

Reply:
See my comment below. I'm +1 without the colors/awk change in Makefile. I'm giving +1, feel free to merge, but please remove the colors logic (we have -1 other changes that did similar things in the past).